src/Pages/Login/Login.scss

```
@@ -0,0 +1,76 @@
+.userLoginForm{
+  margin-top: 0em;
+  
+  .userLoginContainer{
+    border-top: 1px solid #ECECEC;
+    padding: 20px;
+
+    .formLoginItem{
+      margin-bottom: 20px;
+
+      label{
+        display: block;
+        flex-direction: column;
+        color:#000;
+        font-size: 14px;
+        font-weight: bold;
+        cursor: default;
+        
+        span{
+          display:inline;
+        }
+
+        input{
+          display: block;
+          background: none;
+          border: none;
+          border-bottom: solid 2px #000;
+          padding: 0 0 5px 0;
+          transition: border 0.1s;
+          justify-self: center;
+        }
+      }
+
+      
+    }
+
+    .formAction{
+      position: relative;
+
+      .personIcon{
+        font-weight: bold;
+        font-size: 25px;
+        position: absolute;
+        z-index: 0;
+        color: white;
+        margin: 6px 6px;
+      }
+
+      input{
+        background-color: black;
+        border: medium none;
+        border-radius: 0;
+        color:#fff;
+        cursor: pointer;
+        font-size: 16px;
+        font-weight: 600;
+        letter-spacing: 0.25px;
+        margin-right: 10px;
+        text-transform: uppercase;
+        padding: 12px 20px 10px 35px;
+      }
+    }
+
+    .formLink{
+      margin-top: 10px;
+      .linkText{
```

```suggestion
    .formLink {
      margin-top: 10px;\
      
      .linkText {
```
- 클래스명 다음 한칸 띄우고 중괄호 열어주세요
- 새로운 셀렉터 뒤에 한줄 띄워주세요